Features/Contacts/Sources/View/ContactListItemView.swift

```diff
@@ -0,0 +1,48 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+
+import SwiftUI
+import Components
+import Style
+
+public struct ContactListItemView: View {
+    
+    public let name: TextValue?
```

Comment:
```suggestion
    public let name: TextValue?
```

I think name should be non optional?